test/common/http/conn_manager_impl_test_2.cc

```diff
@@ -3878,35 +3878,6 @@ TEST_F(HttpConnectionManagerImplTest, HeaderValidatorRejectTrailersBeforeRespons
   conn_manager_->onData(fake_input, false);
 }
 
-TEST_F(HttpConnectionManagerImplTest, HeaderValidatorFailTrailersTransformationBeforeResponse) {
-  codec_->protocol_ = Protocol::Http11;
-  setup();
-  expectUhvTrailerCheck(HeaderValidator::ValidationResult(
```

Comment:
I have realized that this test had the right name but not the right expectation here that made it duplicated. Can you please change this expectation, instead of deleting the entire test like so:

```
  expectUhvHeaderCheck(
      HeaderValidator::ValidationResult::success(),
      HeaderValidator::TransformationResult(
          HeaderValidator::TransformationResult::Action::Reject,
          "bad_header_map"));
```

Reply:
Thank you for pointing this out! Just updated, PTAL!